vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -858,6 +858,50 @@ testContainerImagePrefetchScript() {
   return 0
 }
 
+testBccTools() {
+
+os="$1"
+executable=0
+not_executable=0
+declare -a notExecutableTools=()
+
+for tool in /usr/share/bcc/tools/*; do
+  if [ -x "$tool" ]; then
+    executable=$((executable + 1))
+  else
+    not_executable=$((not_executable + 1))
+    notExecutableTools+=("$tool")
+    echo "Not Executable: $tool"
+  fi
+done
+
+echo "Total tools: $((not_executable + executable))"
+echo "Executable Tools: $executable"
+echo "Not Executable Tools: $not_executable"
+
+if [ "$os" == "Ubuntu" ]; then
+  if [ "$executable" -ne 127 ] || [ "$not_executable" -ne 2 ]; then
```

Comment:
We should not hardcode these number of tools since that may change based on the version or if we install other tools -- that might lead to manual changes necessary for this test later

Reply:
Can we have a better basic verification of bcc that doesn't actually need counting of installed binaries or at least avoids hardcoded counts?

Reply:
Is it possible to get expected counts programatically before this function is run, and then pass that as input parameters to this function? In that case you can count the expected files as well.